nodes/delete_node.js

```diff
@@ -1,30 +1,87 @@
 module.exports = function (RED) {
+  'use strict';
+
   const { WazoApiClient } = require('@wazo/sdk');
+  
+  function setProperty(node, msg, name, type, value) {
```

Comment:
would be nice to have this function in common file to reuse it.

Reply:
Agreed :+1: